dashboard/src/components/AppView/AppView.tsx

```diff
@@ -231,6 +262,7 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
           break;
         case "Service":
           result.services.push(resource);
+          result.serviceRefs.push(new ResourceRef(resource.item, namespace));
```

Comment:
So the namespace in this case might not be inside the resourceItem? What's `namespace` in this case?

Reply:
The release namespace is used when the resource in the manifest doesn't have a namespace (likely since charts leave the namespace undefined so it deploys in whatever namespace the user chooses). I'll rename this to `releaseNamespace`, didn't want to before since it was already that.